<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUI</class>
 <widget class="QMainWindow" name="MainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QFrame" name="findGroup">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Find...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="resultsLabel">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="closeFind">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="textEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="findPrevB">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="findNextB">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="matchCase">
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>697</width>
     <height>24</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_PDF"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionProperties"/>
   </widget>
   <widget class="QMenu" name="menuPresentation">
    <property name="title">
     <string>&amp;Presentation</string>
    </property>
    <addaction name="actionStart_Begin"/>
    <addaction name="actionStart_Here"/>
    <addaction name="actionStop_Presentation"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionFind"/>
    <addaction name="actionFind_Next"/>
    <addaction name="actionFind_Previous"/>
    <addaction name="actionClearHighlights"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionScroll_Mode"/>
    <addaction name="actionSelect_Mode"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="separator"/>
    <addaction name="actionFirst_Page"/>
    <addaction name="actionPrevious_Page"/>
    <addaction name="actionNext_Page"/>
    <addaction name="actionLast_Page"/>
    <addaction name="separator"/>
    <addaction name="actionBookmarks"/>
    <addaction name="separator"/>
    <addaction name="actionRotate_Counterclockwise"/>
    <addaction name="actionRotate_Clockwise"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPresentation"/>
   <addaction name="menuSettings"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFit_Page"/>
   <addaction name="actionFit_Width"/>
   <addaction name="separator"/>
   <addaction name="actionSingle_Page"/>
   <addaction name="actionDual_Pages"/>
   <addaction name="actionAll_Pages"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_In_2"/>
   <addaction name="actionZoom_Out_2"/>
  </widget>
  <action name="actionOpen_PDF">
   <property name="text">
    <string>Open PDF</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionFit_Page">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fit Page</string>
   </property>
  </action>
  <action name="actionFit_Width">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fit Width</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionSingle_Page">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Single Page</string>
   </property>
  </action>
  <action name="actionDual_Pages">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dual Pages</string>
   </property>
  </action>
  <action name="actionAll_Pages">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>All Pages</string>
   </property>
  </action>
  <action name="actionStart_Begin">
   <property name="text">
    <string>Start Presentation (at beginning)</string>
   </property>
  </action>
  <action name="actionStop_Presentation">
   <property name="text">
    <string>Stop Presentation</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionFind_Next">
   <property name="text">
    <string>Find Next</string>
   </property>
  </action>
  <action name="actionFind_Previous">
   <property name="text">
    <string>Find Previous</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionFirst_Page">
   <property name="text">
    <string>First Page</string>
   </property>
  </action>
  <action name="actionPrevious_Page">
   <property name="text">
    <string>Previous Page</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionNext_Page">
   <property name="text">
    <string>Next Page</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionLast_Page">
   <property name="text">
    <string>Last Page</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionBookmarks">
   <property name="text">
    <string>Bookmarks</string>
   </property>
  </action>
  <action name="actionRotate_Counterclockwise">
   <property name="text">
    <string>Rotate Counterclockwise</string>
   </property>
  </action>
  <action name="actionRotate_Clockwise">
   <property name="text">
    <string>Rotate Clockwise</string>
   </property>
  </action>
  <action name="actionScroll_Mode">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scroll Mode</string>
   </property>
  </action>
  <action name="actionZoom_Out_2">
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionZoom_In_2">
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionStart_Here">
   <property name="text">
    <string>Start Presentation (current slide)</string>
   </property>
  </action>
  <action name="actionSelect_Mode">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select Mode</string>
   </property>
  </action>
  <action name="actionClearHighlights">
   <property name="text">
    <string>Clear Highlights</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
